src/main/java/Cursos.java

```diff
@@ -0,0 +1,97 @@
+package main.java;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet implementation class Cursos
+ */
+@WebServlet("/Cursos")
+public class Cursos extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+	private static final Logger LOGGER = Logger.getLogger("main.java.Login");
+       
+    /**
+     * @see HttpServlet#HttpServlet()
+     */
+    public Cursos() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/manual/cf18patricia.lamadrid/Sudoers-Academy/pullRequest?prid=5438913)